modules/dailyhuntBidAdapter.md

```diff
@@ -0,0 +1,64 @@
+# Overview
+
+```
+Module Name:  Dailyhunt Bid Adapter
+Module Type:  Bidder Adapter
+Maintainer: Dailyhunt
+```
+
+# Description
+
+Connects to dailyhunt for bids.
+
+Dailyhunt bid adapter supports Banner and Native.
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: 'test-banner-adunit-code',
+            sizes: [[300, 250], [320, 50]],
+            bids: [
+                {
+                    bidder: 'dailyhunt',
+                    params: {
+                      partnerId: 'pb-partnerId'
```

Comment:
Your test parameters should return a response from your endpoint. Using this ad unit in a test page, I saw no response from https://money.dailyhunt.in/openrtb2/auction

Reply:
Changes Fixed..

Reply:
Still seeing no response from the server given this banner ad unit

Reply:
![Screenshot from 2019-12-14 10-42-25](https://user-images.githubusercontent.com/21100078/70843837-8546f300-1e5e-11ea-8ae9-fed1006e524a.png)

I created new adunit for display and one for native. And ads are rendered at my end.
can you please confirm the same ?

Reply:
I'll try this. Can you send me the requests that resulted in these ads being rendered?

Reply:
Sure Just give me a minute.
I'll share auction and google request url.

Reply:
**END POINT:** https://money.dailyhunt.in/openrtb2/auction
**HTTP TYPE:** POST
**JSON BODY:**
{
    "id": "cc149d47-359d-494e-a175-a1fab54db0bc",
    "imp": [
        {
            "id": "0",
            "banner": {
                "id": "banner-cc149d47-359d-494e-a175-a1fab54db0bc",
                "format": [
                    {
                        "h": 250,
                        "w": 300
                    },
                    {
                        "h": 50,
                        "w": 320
                    }
                ]
            },
            "bidfloor": 0,
            "ext": {
                "dailyhunt": {
                    "partnerId": "pb-partnerId",
                    "pagetype": "sources",
                    "placementId": 12345,
                    "env": "prod",
                    "zone": "web",
                    "subSlots": "web-3",
                    "pbRequestId": "2862fe3869b49c",
                    "format": "json"
                }
            }
        }
    ],
    "site": {
        "id": "0",
        "page": "http://api.adengine.com:8887/prebid_example_2.html"
    },
    "device": {
        "userAgent": "Mozilla/5.0 (Linux; Android 8.0.0; Pixel 2 XL Build/OPD1.170816.004) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.79 Mobile Safari/537.36"
    },
    "user": {
        "id": ""
    }
}
**HTTP SAMPLE RESPONSE:**
{
    "id": "cc149d47-359d-494e-a175-a1fab54db0bc",
    "seatbid": [
        {
            "bid": [
                {
                    "id": "0",
                    "impid": "0",
                    "price": 1.4,
                    "adm": "<html>\n\t<head></head>\n\t<body>\n\t\t<img class=\"ad_beacon\" style=\"display: none;\" src=\"https://money.dailyhunt.in/impression?uniqueId=9358780685df7a98c89d0c4.17855062&adId=123680&campaignId=70459&adPlacement=web&partnerRef=&city=tapi&state=gujarat&country=india&clientId=935878068\" />\n\t\t<a href=\"https://money.dailyhunt.in/click?clientId=935878068&uniqueId=9358780685df7a98c89d0c4.17855062&adId=123680&campaignId=70459&adPlacement=web&reqTime=1576511884&billingTypeId=3&orderId=1&forceTracker=&partnerRef=&city=tapi&state=gujarat&country=india&clientId=935878068&leadId=&__oadest=https%3A%2F%2Fmontu1996.github.io%2F\" target=\"_blank\" onclick=\"javascript:void(0);\" class=\"direct-anchor\">\n\t\t\t<img src=\"https://acdn.newshuntads.com/daedalus-banners/123680/1574693569_18748_300x250.jpg\" width=\"100%\">\n\t\t</a>\n\t\t<meta name=\"viewport\" content=\"width=device-width, initial-scale=1, user-scalable=no\">\n\t\t<style>body { clear: both; margin: 0; padding: 0; overflow: hidden;}</style>\n\t\t<title></title>\n\t</body>\n</html>",
                    "adid": "123680",
                    "crid": "9358780685df7a98c89d0c4.17855062",
                    "dealid": "9358780685df7a98c89d0c4.17855062",
                    "w": 300,
                    "h": 250,
                    "ext": {
                        "prebid": {
                            "type": "banner"
                        }
                    }
                }
            ],
            "seat": "dailyhunt"
        }
    ],
    "ext": {
        "responsetimemillis": {
            "dailyhunt": 9
        }
    }
}

Reply:
**GOOGLE REQUEST URL:**
https://securepubads.g.doubleclick.net/gampad/ads?gdfp_req=1&pvsid=221974338596427&correlator=3790886290959736&output=ldjh&impl=fifs&adsid=ChAIgPPc7wUQytW-soi18sd1EjsAEC-N0g17A0kLBXIGncU7nrnbUJCTZspKTniwFdD_D6ol__5yguTnXi1W_5zk689BP6VuWs4OVLhe5A&jar=2019-12-16-12&eid=21065298%2C21063205%2C21063636&vrg=2019121002&guci=2.2.0.0.2.2.0.0&sc=0&sfv=1-0-37&ecs=20191216&iu_parts=83414793%2Cprebid_test_display&enc_prev_ius=%2F0%2F1&prev_iu_szs=300x250&prev_scp=hb_format_dailyhunt%3Dbanner%26hb_source_dailyhunt%3Dclient%26hb_size_dailyhunt%3D300x250%26hb_pb_dailyhunt%3D1.40%26hb_adid_dailyhunt%3D5715bd3891cac9%26hb_bidder_dailyhunt%3Ddailyhunt%26hb_format%3Dbanner%26hb_source%3Dclient%26hb_size%3D300x250%26hb_pb%3D1.40%26hb_adid%3D5715bd3891cac9%26hb_bidder%3Ddailyhunt&eri=1&cookie=ID%3Ddc47f750c2228b82%3AT%3D1574770048%3AS%3DALNI_MY26chxM191k7re6Zbug3J88ZDMGg&bc=23&abxe=1&lmt=1576512052&dt=1576512052794&dlt=1576512051996&idt=523&frm=20&biw=411&bih=823&oid=3&adxs=8&adys=95&adks=3132071267&ucis=1&ifi=1&u_tz=330&u_his=3&u_h=823&u_w=411&u_ah=823&u_aw=411&u_cd=24&u_sd=3.5&flash=0&url=http%3A%2F%2Fapi.adengine.com%3A8887%2Fprebid_example_2.html&dssz=10&icsg=2216&std=0&vis=1&scr_x=0&scr_y=0&psz=395x792&msz=395x250&ga_vid=1211466752.1576512053&ga_sid=1576512053&ga_hid=1832887449&fws=0&ohw=0

Reply:
Thanks. I'm able to see the ad rendering in my test page now. But, based on the response headers from `openrtb2/auction`, doesn't look like the response is compressed.

http://prebid.org/dev-docs/bidder-adaptor.html#bidder-adaptor-Required-Adapter-Conventions

Reply:
@msm0504 issues is fixed now.
can you please check once ?

![Screenshot from 2019-12-17 00-54-26](https://user-images.githubusercontent.com/21100078/70936253-edb4f080-2067-11ea-91f9-4415bdfe79c6.png)

Thanks

Reply:
Looks good. Thanks.